index.html

```diff
@@ -176,50 +241,80 @@ <h3 id="matters-considered-by-the-placement-rules">Matters considered by the pla
+    <span class="checkme">
+    Specifically, the ruby string which protrudes from the base character 
+    string is not allowed to be laid over the characters preceding or 
+    following, nevertheless preceding or following characters are Kanji or 
+    Kana characters. 
+    </span>
+
+    <aside class=note title="Protruding over surrounding characters">
+      The main placement method defined in JIS X 4051 [[JISX4051]]
+      allows some amount of overhang over the preceding and following base characters,
+      but recognizes the method defined here as an allowed variant.
     </aside>
   </li>
-  <li>The method detailed in this document attempts to reduce exceptions as much as possible.
-    Therefore, there is no requirement for complex processing.
   <li>The method is agnostic to horizontal vs vertical writing,
-    and will use the same logic in either case.</li>
-  <li>The method places the ruby string relative to the base character string
-    the same way when they occur in the middle, start, or end of the line.
-    Moreover, this method does not change the relative position
-    of the ruby string to the base character string
-    depending on preceding or subsequent characters.
-    In other words, this method calculates a position
-    for the ruby relative to the base string
-    that does not change depending on context.</li>
-  <li>Generally speaking, the processing method is based on JIS X 4051 [[JIS4051]]
-    (Formatting rules for Japanese documents).
-    However, in some cases, optional steps are used.</li>
-  <li>The ruby font size is set to half of the base character’s size as a default.
-    However, the method supports using different sizes than 1/2.
-    <aside class=note title="Reference size">
-      Because the size of ruby characters used in JIS X 4051 [[JIS4051]] set to 1/2
-      there are many examples that use the size of the ruby character as the reference.
-      However, since ruby is not restricted to 1/2,
-      this document uses the size the base character as the reference.
-    </aside>
-    <aside class=note title="Size of ruby characters">
-      In movable type typesetting,
-      when ruby characters of size 3.5 points were not available,
-      based characters of 7 points where sometimes paired
-      with ruby characters of 4 points.
-      Also, ruby associated with large base characters,
-      such as those in titles,
-      are sometimes smaller than 1/2.
-    </aside>
-  </li>
-  <li>While there are cases of ruby on both sides of the base string exist,
-    the method defined here only handles ruby on one side.
-    Handling both sides is left as a future exercise.</li>
+    and will use the same logic in either case.
+    <spanclass="checkme">
+    Specifically, the center of the ruby string and of the base character 
+    string are aligned in the inline direction for mono-ruby. 
+    </span>
+    </li>
+  <li><span class="checkme">Two step processing method is taken. 
```

Comment:
Two step -> "Two-step"?

Reply:
I actually thought of that,,, but it's just a local definition which is not used so often, also no other document-wide common processing methods. Do you think this is worth to have dfn and to make as such defined word?

I'd also /cc @kidayasuo

Reply:
+1 to "two-step". Compare with "the process is two step**s**.

Reply:
done. without dfn.